management/clipper_manager.py

```diff
@@ -594,28 +596,67 @@ def centered_predict(inputs):
         if not os.path.exists(serialization_dir):
             os.makedirs(serialization_dir)
 
-        # Write out function serialization
-        func_file_path = "{dir}/{predict_fname}".format(
-            dir=serialization_dir, predict_fname=predict_fname)
-        with open(func_file_path, "w") as serialized_function_file:
-            serialized_function_file.write(serialized_prediction_function)
-        print("Serialized and supplied predict function")
-
         # Export Anaconda environment
-        subprocess.call(
+        process = subprocess.Popen(
             "PIP_FORMAT=legacy conda env export >> {environment_fname}".format(
                 environment_fname=environment_fname),
             shell=True)
+        process.wait()
```

Comment:
We want to make sure the environment file gets written out before we check it for problems